src/book.py

```diff
@@ -56,23 +56,23 @@ def create_operation(
         coin: str,
         row: int,
         file_path: Path,
-    ) -> Optional[tr.Operation]:
+    ) -> Type[tr.Operation]:
 
         try:
             Op = getattr(tr, operation)
```

Comment:
Mypy does not know what type getattr returns. Add an assert isinstance or in this case. Assert issubclass (dunno how the function is named. You might have to Google for this)